static-assets/components/cstudio-admin/mods/admin-configurations.js

```diff
@@ -534,12 +544,32 @@ function moduleLoaded() {
             showErrorDialog(errMessage);
           }
 
-        } catch(e) {
+        } catch (e) {
           showErrorDialog(e.message);
         }
 
       });
 
+      // Styles are inline because the dialog its not at admin-config container level.
+      $('#encryptHint').click((e) => {
+        e.preventDefault();
+        CStudioAuthoring.Operations.showSimpleDialog(
+          'error-dialog',
+          CStudioAuthoring.Operations.simpleDialogTypeINFO,
+          formatMessage(adminConfigurationMessages.encryptMarked),
+          '<div style="display: flex;">' +
+          '<i class="fa fa-info-circle" aria-hidden="true" style="font-size: 20px; color: #5cc7fa; margin-right: 10px;"></i>' +
+          `<p style="padding: 0; margin: 0">${formatMessage(adminConfigurationMessages.encryptHint)}</p></div>`,
+          [{
```

Comment:
Format properly. Though I reckon the ok button is what the show dialog does automatically.